products/oracle-apex.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Oracle APEX
+category: framework
+tags: oracle
+iconSlug: oracle
+permalink: /oracle-apex
+alternate_urls:
+-   /orclapex
+versionCommand: SELECT VERSION_NO FROM APEX_RELEASE;
+releasePolicyLink: https://www.oracle.com/database/technologies/appdev/apex/collateral/#assistance
+releaseColumn: false
+releaseDateColumn: true
+extendedSupportColumn: false
+activeSupportColumn: false
+
+releases:
+-   releaseCycle: "23.2"
+    # https://blogs.oracle.com/apex/post/announcing-oracle-apex-232
+    releaseDate: 2023-11-14
+    eol: 2025-05-01
+    link: https://apex.oracle.com/en/platform/features/whats-new-232/
+
+-   releaseCycle: "23.1"
+    # https://blogs.oracle.com/apex/post/announcing-oracle-apex-231
+    releaseDate: 2023-05-17
+    eol: 2024-12-01
+    link: https://apex.oracle.com/en/platform/features/whats-new-231/
+
+-   releaseCycle: "22.2"
+    # https://blogs.oracle.com/apex/post/announcing-oracle-apex-222
+    releaseDate: 2022-11-10
+    eol: 2024-12-01
+    link: https://apex.oracle.com/en/platform/features/whats-new-222/
+
+-   releaseCycle: "22.1"
+    # https://blogs.oracle.com/apex/post/announcing-oracle-apex-221
+    releaseDate: 2022-05-03
+    eol: 2024-11-01
```

Comment:
```suggestion
    eol: 2024-12-01
```

According to https://www.oracle.com/database/technologies/appdev/apex/collateral/#assistance.